### DISCUSSION TITLE
Edge Functions: Deploy More Functions at No Extra Cost

### DISCUSSION THREAD

**Discussion by kevcodez** [supabase/supabase#28062] [Changelog]
In an effort to simplify pricing, we are going to remove usage-based billing for the number of Edge Functions in your projects. Instead, we are going for a bigger quota across all plans at no extra costs. We picked the limits to ensure all customers are benefiting from this change.

Free Plan customers can now create 25 instead of 10 functions without the need to upgrade to a paid Plan.

| | Free Plan  | Pro Plan | Team Plan | Enterprise Plan
| ------------- | ------------- | ------------- | ------------- | ------------- |
| Before  | 10 included | 100 included, then $10 per additional 100  | 100 included, then $10 per additional 100  | Custom  |
| After  | 25 included  | 500 included  | 1000 included  | Unlimited

This change is effective immediately and in case you were previously exceeding the number of included functions on a paid Plan, you will no longer be charged for it.

**Comment by nukemonk** [comment]
It seems like that [supabase.com/pricing](https://supabase.com/pricing) page hasn't been updated for the free plan only. It still shows 10 functions.

**Reply by kevcodez** [reply]
Thanks for pointing it out - we'll fix it right away

**Reply by kevcodez** [reply]
https://github.com/supabase/supabase/pull/28684
